README.md

```diff
@@ -18,33 +36,29 @@ This new Custom role along with the built-in "Reader" role is applied to the sub
 3. Run the following
 ```bash
 curl -s https://raw.githubusercontent.com/cloudconformity/temp-azure-custom-policy/master/apply-roles | bash /dev/stdin \
-    --application-id <App registration client id> \
-    --subscription-id <Subscription id>
+    --application-id fj63c3b9-46e3-41ca-bf79-344a076fa9b2
```

Comment:
❓ I don't really have any context on the azure stuff, but are we ok exposing this key? to the public

Reply:
This id is just a made up as an example, but I have removed it to avoid confusion as a user might just have copy and pasted it thinking it is important to be that id